syne_tune/tuner.py

```diff
@@ -587,14 +593,20 @@ def _update_running_trials(
                     or done_trials[trial_id][1] != Status.paused
                 ):
                     logger.info(f"Trial trial_id {trial_id} completed.")
+                # If scheduler marks trial as `Status.paused`, this must not be
+                # flipped back to `Status.completed`
```

Comment:
I suppose it is the other way around:
```suggestion
                # If scheduler marks trial as `Status.completed`, this must not be
                # flipped back to `Status.paused`
```